### DISCUSSION TITLE
Duplicate fantom windows created by using macOS native fullscreen

### DISCUSSION THREAD

**Discussion by SylvanFranklin** [nikitabobko/AeroSpace#956] [potential-bugs]
**Describe the bug**

Whenever I use the shortcut `macos-native-fullscreen`, I get a duplicate window in the background. I'm on the latest stable version and macOS Sonoma 14.3.1. I first noticed this thanks to my transparent terminal, but it also effects firefox and other desktop apps. These "Fantom" instances can not be clicked on or otherwise interacted with, but they can be quit with command+q

https://github.com/user-attachments/assets/75895d6c-9fbb-45fb-ab40-4b3d5894e9ce

**I could not find duplicate**

**Comment by kenkrik** [comment]
I have also noticed this happening lately.

**Comment by SylvanFranklin** [comment]
This seems to only create a window in the place of the old window. If I use fullscreen and then spawn the window with another one that is not transparent, the duplicate window is hidden. Not sure if that is important or not.
